absl/container/internal/hash_function_defaults.h

```diff
@@ -56,7 +56,7 @@
 #include "absl/container/internal/common.h"
 #include "absl/hash/hash.h"
 #include "absl/meta/type_traits.h"
-#include "absl/strings/cord.h"
+#include "absl/strings/internal/cord_lookup_support.h"
```

Comment:
This doesn't work as is since the Bazel build now has an undeclared dependency on this header. The same goes for the other places where the dependency was removed.